patches/exploding_kings.cpp

```diff
@@ -56,6 +56,12 @@ void buildCountdownToSaboteur(std::vector<int16_t> &COUNTDOWN_UNITS, genie::Civ
               << std::endl;
 }
 
+void patchMonumentToKeepItFromExploding(genie::Civ &civ) {
+    genie::Unit &monument = civ.Units.at(ID_MONUMENT);
+    monument.HitPoints = 30000; //Max HP allowed in engine with some buffer so not to risk overflow
```

Comment:
changing hp to -1 and blood unit ID to saboteur/tristan here is really funny for April fools